src/test/java/redis/clients/jedis/tests/JedisSentinelPoolWithCompleteCredentialsTest.java

```diff
@@ -68,67 +60,92 @@ public void repeatedSentinelPoolInitialization() {
+
+      JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, sentinels, poolConfig, masterConfig, sentinelConfig);
       pool.getResource().close();
       pool.destroy();
     }
   }
 
   @Test(expected = JedisConnectionException.class)
   public void initializeWithNotAvailableSentinelsShouldThrowException() {
-    Set<String> wrongSentinels = new HashSet<String>();
-    wrongSentinels.add(new HostAndPort("localhost", 65432).toString());
-    wrongSentinels.add(new HostAndPort("localhost", 65431).toString());
 
-    JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, wrongSentinels);
+    GenericObjectPoolConfig<Jedis> poolConfig = new GenericObjectPoolConfig<>();
+
+    JedisClientConfig masterConfig = DefaultJedisClientConfig.builder()
+        .connectionTimeoutMillis(1000).socketTimeoutMillis(1000).database(2)
+        .user("acljedis").password("fizzbuzz").build();
+
+    JedisClientConfig sentinelConfig = DefaultJedisClientConfig.builder()
+        .connectionTimeoutMillis(1000).socketTimeoutMillis(1000)
+        .user("default").password("foobared").build();
+
+    JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, sentinels, poolConfig, masterConfig, sentinelConfig);
+    pool.getResource().close();
     pool.destroy();
   }
 
   @Test(expected = JedisException.class)
   public void initializeWithNotMonitoredMasterNameShouldThrowException() {
-    final String wrongMasterName = "wrongMasterName";
-    JedisSentinelPool pool = new JedisSentinelPool(wrongMasterName, sentinels);
+
+    GenericObjectPoolConfig<Jedis> poolConfig = new GenericObjectPoolConfig<>();
+
+    JedisClientConfig masterConfig = DefaultJedisClientConfig.builder()
+        .connectionTimeoutMillis(1000).socketTimeoutMillis(1000).database(2)
+        .user("acljedis").password("fizzbuzz").build();
+
+    JedisClientConfig sentinelConfig = DefaultJedisClientConfig.builder()
+        .connectionTimeoutMillis(1000).socketTimeoutMillis(1000)
+        .user("sentinel").password("foobared").build();
+
+    JedisSentinelPool pool = new JedisSentinelPool("wrongMasterName", sentinels, poolConfig, masterConfig, sentinelConfig);
+    pool.getResource().close();
     pool.destroy();
   }
 
   @Test
   public void checkCloseableConnections() throws Exception {
     GenericObjectPoolConfig<Jedis> config = new GenericObjectPoolConfig<>();
 
-    JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, sentinels, config, 1000, "default",
-        "foobared", 2);
+    JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, toStrings(sentinels), config,
+        1000, 1000, "acljedis", "fizzbuzz", 2, null, 1000, 1000, "sentinel", "foobared", null);
     try (Jedis jedis = pool.getResource()) {
-      jedis.auth("default", "foobared");
       jedis.set("foo", "bar");
       assertEquals("bar", jedis.get("foo"));
     }
     pool.close();
     assertTrue(pool.isClosed());
   }
 
-  @Test
-  public void ensureSafeTwiceFailover() throws InterruptedException {
```

Comment:
We need two more (four in total) Redis+Sentinel instances just to support this single test. But exact test is also present in JedisSentinelPoolTest and JedisSentinelTest. So I took the chance of removing this test. Also note that only this test is removed in this entire PR.